all.json

```diff
@@ -32,5 +32,8 @@
     "polkawallets.site",
     "polkodot.network",
     "walletsynchronization.com"
+    "polkadot-distribution.live"
```

Comment:
```suggestion
    "polkadot-distribution.live",
```

Also needs a `,` at the end of the previously line.

Reply:
new to this, have to learn the details. i changed it and updated

Reply:
No issues at all.